igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/github/client/GitHubMaster.groovy

```diff
@@ -16,53 +16,63 @@
 
 package com.netflix.spinnaker.igor.scm.github.client
 
-import com.netflix.spinnaker.igor.config.GitHubProperties
+
 import com.netflix.spinnaker.igor.scm.AbstractScmMaster
-import com.netflix.spinnaker.retrofit.Slf4jRetrofitLogger
-import org.springframework.context.annotation.Bean
-import retrofit.Endpoints
-import retrofit.RequestInterceptor
-import retrofit.RestAdapter
-import retrofit.client.OkClient
-import retrofit.converter.JacksonConverter
+import com.netflix.spinnaker.igor.scm.github.client.model.GetRepositoryContentResponse
+import com.netflix.spinnaker.kork.web.exceptions.NotFoundException
+import groovy.util.logging.Slf4j
+import retrofit.RetrofitError
 
-import javax.validation.Valid
+import java.util.stream.Collectors
 
 /**
  * Wrapper class for a collection of GitHub clients
  */
-class GitHubMaster  extends AbstractScmMaster {
```

Comment:
since it's in OSS and we don't really know who is using this class, can it be backward-incompatible?

Reply:
The current `GitHubMaster` is just a copy of `GitHubConfig`, that defines the `GitHubMaster` bean, which is not used, and it didn't implement any methods from `AbstractScmMaster`, so shouldn't have backward compatibility issues.